partiql-spi/src/main/java/org/partiql/spi/value/Datum.java

```diff
@@ -736,18 +727,28 @@ static Datum blob(@NotNull byte[] value, int length) {
     // DATE/TIME
 
     @NotNull
-    static Datum date(@NotNull Date value) {
+    static Datum date(@NotNull LocalDate value) {
```

Comment:
Perhaps I'm missing some context for the changes here. I had a couple questions
1. Why are we opting to use the Java-defined datetime classes rather than the currently-defined datetime classes/interfaces in SPI? I'm not opposed but just would like some more context here.
2. Will those datetime classes/interfaced defined in SPI be deleted at some point?